src/app/plugins/platform/Docker/plugins/cms/Drupal8/installDrupal.ts

```diff
@@ -27,6 +36,12 @@ async function replaceIn(
   return writeFile(path, contents.replace(search, replacement));
 }
 
+/**
+ * An interface for required Drupal project configuration options.
+ *
+ * @export
+ * @interface InstallDrupalOptions
```

Comment:
These two tags aren't needed - they're already in the source code. TypeDoc doesn't have a use for them, and they end up as these weird pills in the output:

![image](https://user-images.githubusercontent.com/194893/63042970-303ba600-be99-11e9-8381-1ac4dd04f95e.png)